package.json

```diff
@@ -45,19 +43,20 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "bl": "^1.1.2",
+    "async": "^2.1.4",
+    "execa": "^0.5.0",
     "go-ipfs-dep": "0.4.4",
     "ipfs-api": "^12.0.3",
     "multiaddr": "^2.1.0",
     "once": "^1.4.0",
     "rimraf": "^2.5.4",
-    "run-series": "^1.1.4",
-    "shutdown": "^0.2.4",
-    "subcomandante": "^1.0.5"
```

Comment:
@jbenet invested a lot of time making sure we had a thing that would not leave zombie IPFS processes in user machines. It was what blocked https://github.com/ipfs/js-ipfsd-ctl/pull/89 all this time.

Until we have comprehensive tests to make sure that you can spawn a bunch of daemons and make sure they don't become zombies, we can't move to execa. Historically, a way to test this was with `station`

If I understand correctly, migrating to `execa` is not a requirement for any particular feature.